### DISCUSSION TITLE
ZS5101 device: inverted value for safe and usafe

### DISCUSSION THREAD

**Discussion by zanna1972** [zwave-js/zwave-js#6585] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
I have a ZS5101 device (firmware updated) and correctly included in Home assistant.
The entity binary_sensor entity, corresponding to the "safety" of the device, is "unsafe" when the device is ok, and "safe" when I shake the device.

What did you expect to happen instead?
When I shake the device I expect the entity became "unsafe". When I stop shaking I expect the entity became "safe".

  2. Click on '...'
  3. Scroll down to '...'

Anything else we should know?

### Software versions

Driver (node-zwave-js): ...

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: 0.4.2

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: Vision Security
Model name: ZS5101
Node ID: 25

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[error_log.txt](https://github.com/zwave-js/node-zwave-js/files/13814269/error_log.txt)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
Unfortunately, we can't do anything about devices that report their status incorrectly. Maybe you can build a template sensor in HA to correct this. I don't know how though.

Your log shows in the end that the "Home Security" sensor status goes from "Intrusion" to idle (at 2024-01-02 23:07:46). Not sure if this is the intended state or not? This is the only change I see in there.
